### PR TITLE
Be 1 init application

### PR DESCRIPTION
**Pull request by viktoriya-okseniuk**

### CODE REVIEW

File: README.md
Comment (Ivanus-Volodymyr) [inline README.md:1]: need a description for .env

File: src/app.controller.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
+
+  @Get()
+  healthChecker(): object {
```
Comment (Ivanus-Volodymyr) [inline src/app.controller.ts]: add async and types
object is not good decision

File: src/app.service.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  healthCheck(): object {
```
Comment (Ivanus-Volodymyr) [inline src/app.service.ts]: add async and types
object is not good decision

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

File: src/app.service.ts
Comment (Ivanus-Volodymyr) [inline src/app.service.ts:1]: Very complicated logic for checking server status

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

File: src/app.service.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+export class AppService {
+  async healthCheck(): Promise<string> {
+    interface objTypes {
+      status_code: number;
+      detail: string;
+      result: string;
+    }
+    const obj: objTypes = { status_code: 200, detail: 'ok', result: 'working' };
+    return JSON.stringify(obj);
+  }
```
Comment (Ivanus-Volodymyr) [inline src/app.service.ts]: check in the task what the form of the server status response should be  
do not use JSON.stringify(obj)

also do not forget change types

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

File: src/app.service.ts (hunk 3 above)
Comment (viktoriya-okseniuk) [reply src/app.service.ts]: Please check if i done it well

File: src/app.controller.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { Controller, Get } from '@nestjs/common';
+import { AppService } from './app.service';
+
+@Controller()
+export class AppController {
+  constructor(private readonly appService: AppService) {}
+
+  @Get()
+  async healthChecker(): Promise<string> {
+    const result: string = await this.appService.healthCheck();
```
Comment (Ivanus-Volodymyr) [inline src/app.controller.ts]: make it in one line

File: src/app.service.ts (hunk 3 above)
Comment (Ivanus-Volodymyr) [reply src/app.service.ts]: do not use template string
check task please

**Review by Ivanus-Volodymyr — CHANGES_REQUESTED** [review]

**Pull request merged**